### DISCUSSION TITLE
cgpt on archlinux

### DISCUSSION THREAD

**Discussion by omansh-krishn** [sebanc/brunch#1644] [General]
Hello!
I am on Arch linux, and the 'cgpt' package is not available in arch repository.
So I installed it from AUR(arch user repository), and its name was cgpt-bin (https://aur.archlinux.org/packages/cgpt-bin). In the pkgbuild file it defined a link to ubuntu's cgpt package path, which is no longer available (the version of pakage is not available) ( I cheked it by searching for the file on the ubuntu repo).
pkg build file--
'''
pkgname=cgpt-bin
pkgver=13597.B
pkgrel=1
_rel=R88
pkgdesc="GPT manipulation tool with support for Chromium OS extensions"
arch=('x86_64')
url="https://chromium.googlesource.com/chromiumos/platform/vboot_reference/"
license=('BSD')
depends=('glibc' 'libuuid.so')
provides=('cgpt')
source=("cgpt.deb::http://archive.ubuntu.com/ubuntu/pool/universe/v/vboot-utils/cgpt_0~${_rel}-${pkgver}-${pkgrel}_amd64.deb"")
sha512sums=("d784e09009855eb28985503dd2c2b32fee6db640d0e177db9fd534de0adc10a4e7773f9dd8a754f8a523f87754d2523707c25eb675ed61e1302b7eee78f65ce8")

package() {
	cd "$srcdir"
	bsdtar -x -f data.tar.xz -C "$pkgdir"
	cd "$pkgdir"
	install -Dm644 ./usr/share/doc/cgpt/copyright "$pkgdir"/usr/share/licenses/cgpt/COPYRIGHT
	rm -r ./usr/share/doc
}
'''
According to it, it must download the file -- 
http://archive.ubuntu.com/ubuntu/pool/universe/v/vboot-utils/cgpt_0~R88--13597.B-1_amd64.deb"

which is no longer available on the server. I changed the variables to get the latest cgpt package, but it gave me errors.

I found the same version of file on debian server, so I changed it's url to ---
http://ftp.de.debian.org/debian/pool/main/v/vboot-utils/cgpt_0~R88-13597.B-1_amd64.deb
(I also had to change the value for sha512 as it was different for the package downloaded from debian's server)
the installation was also successful, but when I run the command --
sudo bash chromeos-install.sh -src chromeos_filename.bin -dst ~/tmpmount/chromeos.img -s size 
( I had changed the variables when I ran it on my pc.)
I am still getting error that
'''which: no cgpt in (/usr/local/bin:/usr/bin:/usr/local/sbin:/var/lib/flatpak/exports/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
cgpt needs to be installed first
'''

Also, I am new to arch...

**Comment by sebanc** [comment]
IIRC the correct package in AUR is vboot-utils

**Comment by omansh-krishn** [comment]
I tried that too, but it needed a dependency called 'trousers', which too was in aur itself. I installed that too, but then too it wasn't working.
Meanwhile, I used DEBTAP to convert cgpt .deb to .zst and installed it. It worked!!
It just asked for one more pakage --''pv'', which was available in arch repo.I installed it and had done the further installation.
Sent this from ChromeOS itself.

**Comment by kRHYME7** [comment]
Hello as of 7/25/2023 yay works


yay -S cgpt

**Comment by secext2022** [comment]
We can use `toolbox`: <https://containertoolbx.org/>

Then we can create a ubuntu container and use the tools under ArchLinux !
This method is very fast and simple.
